py3status/modules/xrandr_toggle.py

```diff
@@ -88,7 +88,7 @@ def _xrandr(self):
                         monitors.append(source)
             return monitors
         except:
-            return self._error('error while parsing xrandr', None)
+            return self._error('error while parsing xrandr')
```

Comment:
> Wouldn't that make it easier to catch an error from there?

My view is we should not catch it anywhere.  It should just break.  The parsing is fairly fundamental, and you can make it robust enough to not have any errors.